Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM node:18.14.2-slim
+FROM node:18.15.0
 
 WORKDIR /usr/src/app
 COPY package*.json *.js ./
 RUN npm ci --omit=dev
 
 COPY *.js *.json *.sh ./
 CMD [ "npm", "start", "--silent"]
+
+HEALTHCHECK --interval=60s --timeout=1s --start-period=5s --retries=3 CMD [ "curl" "https://127.0.0.1:9999" "--fail" ]
```

Comment:
this is a serving suggestion. I will be configuring this in ecs / terraform explicitly.

> The Amazon ECS container agent only monitors and reports on the health checks specified in the task definition. Amazon ECS does not monitor Docker health checks that are embedded in a container image and not specified in the container definition. Health check parameters that are specified in a container definition override any Docker health checks that exist in the container image.
– https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html